zipkin/src/main/java/zipkin/storage/QueryRequest.java

```diff
@@ -178,7 +178,7 @@ public static Builder builder() {
     private Long minDuration;
     private Long maxDuration;
     private Long endTs;
-    private Long lookback;
+    private Long lookback = 86400000L; // see ZipkinQueryApiV1.defaultLookback
```

Comment:
I can try and help with this part. thx for the heads-up

On Mon, Sep 5, 2016 at 4:53 AM, mck <notifications@github.com> wrote:

> In zipkin/src/main/java/zipkin/storage/QueryRequest.java
> <https://github.com/openzipkin/zipkin/pull/1252#discussion_r77458464>:
>
> > @@ -178,7 +178,7 @@ public static Builder builder() {
> >      private Long minDuration;
> >      private Long maxDuration;
> >      private Long endTs;
> > -    private Long lookback;
> > +    private Long lookback = 86400000L; // see ZipkinQueryApiV1.defaultLookback
>
> I've removed it, and added it to the tests in zipkin-storage/cassandra-3
> But it'll be a performance problem for tests, as many of the tests are in
> SpanStoreTest and SpanCassandraTest.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openzipkin/zipkin/pull/1252/files/248a42064098a4aee21934d09739def2cc2331ad#r77458464>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD61885Yefz0giH5aMhMLd8liUnTWBFks5qmy_fgaJpZM4Jrhum>
> .
>